PSIPro.ino

```diff
@@ -1924,8 +1936,7 @@ void StarWarsIntro(unsigned long time_delay, uint8_t loops, CRGB color, unsigned
     globalPatternLoops = loops;
     if ((runtime != 0) && (!timingReceived)) set_global_timeout(runtime);
     if (timingReceived) set_global_timeout(commandTiming);
-    ledPatternState = 2;
-    // Clear the display the first time through
+    ledPatternState = 0;
```

Comment:
The goal here of not starting on Row 5 was to prevent the "scrolling text" from starting narrow, getting wider, then becoming narrower again.